packages/server/src/environment.ts

```diff
@@ -114,6 +114,7 @@ const environment = {
     DEFAULTS.JS_RUNNER_MEMORY_LIMIT,
   LOG_JS_ERRORS: process.env.LOG_JS_ERRORS,
   DISABLE_USER_SYNC: process.env.DISABLE_USER_SYNC,
+  DISABLE_CONTENT_SECURITY_POLICY: process.env.DISABLE_CONTENT_SECURITY_POLICY,
```

Comment:
Don't need to duplicate this - if its something that is common across services its fine for it to just be in `backend-core/src/environment.ts`